spot-client/src/common/backend/SpotBackendService.js

```diff
@@ -133,9 +133,9 @@ export class SpotBackendService extends Emitter {
      */
     _maybeClearRegistration(error) {
         if (this.isUnrecoverableRequestError(error)) {
+            this.getJwt() && logger.log('Cleared backend registration');
```

Comment:
Would it be better to always log but include if there was a jwt? Same really with any logs? Always log but include data about the log?

Reply:
This one about cleared backend registration is very confusing to me when looking through the logs.